source/documentation/rules-b.html.md.erb

```diff
@@ -0,0 +1,668 @@
+---
+title: NCTS Phase 5 Technical Interface Specification
+weight: 5
+description: Software developers, designers, product owners or business analysts. Integrate your software with the ERMIS service
+---
+#Rules B
+Based on document version 5.15.0-v0.10 and issue date 04/03/2022
+<%= partial 'documentation/partials/rulesintro' %>
+##B1000
+
+<b>Functional Description</b>
+
+Export followed by Transit & Groupage:<br/><br/>During the Transitional Period, in case of “Export Followed by Transit” with multiple export declarations covered by one standard transit declaration (i.e. not a pre-lodged transit declaration), the D.G. PREVIOUS DOCUMENT at CONSIGNMENT ITEM level can include the MRN of the related Export declaration (maximum one Export MRN included per one Consignment item).
+
+<b>Technical Description</b>
+
+N/A<br/>
+
+##B1003
+
+<b>Functional Description</b>
+
+In case of 'Road Transport' if the road vehicle (and/or its road trailers) are different to those declared in the initial declaration message, then they will be reported as discrepancies and not as new transport means.
+
+<b>Technical Description</b>
+
+N/A
+
+##B1016
+
+<b>Functional Description</b>
+
+During the Transitional Period, the data item “Customs office at border reference number”, is required for ‘native CC015C’ and ‘native CC013C’ and optional if CC013C or CC015C is the outcome of the upgraded CC013B or CC015B respectively. After the end of the Transitional Period, the data item will be always required.
+
```

Comment:
Grammar/style issues:
- Remove unnecessary comma before "is required".
- Insert "but is" before "optional".
- Insert comma before "respectively".